### ISSUE TITLE
add points directly by clicking on the graph

### ISSUE THREAD

**Issue by nhammad** [MenesesGHZ/polynomial-regression-js#2]
Currently, we click on points on the grid and then they get added to the graph. Is there any way we can modify this such that we can add points by directly clicking on the graph itself?

**Comment by MenesesGHZ** [comment]
You mean, drawing multiple random points inside of certain ratio with just one click ? If that is the case, it can be done. If you have a different idea please let me know.

**Comment by MenesesGHZ** [comment]
Yes, it can be done by just adding to the Graph.js component the same logic as the Drawer.js component.

**Comment by MenesesGHZ** [comment]
Drawer.js logic: 
https://github.com/MenesesGHZ/polynomial-regression-js/blob/f656958d54a78a6b90665472a2d396a59379cee0/src/components/Drawer.js#L25-L30
https://github.com/MenesesGHZ/polynomial-regression-js/blob/f656958d54a78a6b90665472a2d396a59379cee0/src/components/Drawer.js#L36-L40
